cayleypy/graphs_lib_test.py

```diff
@@ -115,10 +115,32 @@ def test_pancake():
     assert graph.generator_names == ["R1", "R2", "R3", "R4", "R5"]
     assert np.array_equal(
         graph.generators,
-        [[1, 0, 2, 3, 4, 5], [2, 1, 0, 3, 4, 5], [3, 2, 1, 0, 4, 5], [4, 3, 2, 1, 0, 5], [5, 4, 3, 2, 1, 0]],
+        [
+            [1, 0, 2, 3, 4, 5],
+            [2, 1, 0, 3, 4, 5],
+            [3, 2, 1, 0, 4, 5],
+            [4, 3, 2, 1, 0, 5],
+            [5, 4, 3, 2, 1, 0],
+        ],
     )
 
 
+def test_lsl_cycles():
+    graph = PermutationGroups.lsl_cycles(3, add_inverses=True)
+
+    expected_generators = [
+        [1, 2, 0],  # L
+        [0, 2, 1],  # S
+        [2, 0, 1],  # L_inv
+        [0, 2, 1],  # S_inv (совпадает с S для n=3)
+    ]
+    assert all(any(np.array_equal(g, e) for g in graph.generators) for e in expected_generators)
+
+    assert graph.n_generators == 4
```

Comment:
There are in fact 3 generators, because all generators must be distinct. In this case S_inv=S, so S_inv will not be added.

Please correct expectation to match the actual result.

Also you may add a test for n=4, where there will be 4 generators.